src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let copyState = { ...state };
```

Comment:
Consider renaming `copyState` to something more descriptive like `currentState` to better convey its purpose as the current state being transformed.